README.md

```diff
@@ -85,16 +128,33 @@ docker run -d -p 80:8080 \
   presenter:dev script/dev
 ```
 
-### Submitting Content
+### Submitting content
 
-Now the site is running, but you don't have any content submitted, yet. To add some, run the appropriate `script/add-*` script with the path to your clone of a local content repository.
+Now the site is running, but you don't have any content submitted,
+yet. To add some, run the appropriate `script/add-*` script with the
+path to your clone of a local content repository.
 
 ```bash
+# add the control repository:
+script/add-assets ~/writing/drc/deconst-docs-control
```

Comment:
I'd mention that `add-assets` must happen first if you expect the site to render with all of the styling intact. The mappings should appear just fine without it, though.